src/inc/corinfo.h

```diff
@@ -1686,6 +1686,8 @@ struct CORINFO_CALL_INFO
     };
 
     CORINFO_CONST_LOOKUP    instParamLookup;    // Used by Ready-to-Run
+
+    BOOL                    secureInvoke;
```

Comment:
> Is this a per-call property or a per-method property?
This is a per-method property, but related with the caller (not the callee). This is required for all the delegate invoke call inside secure delegate invoke method (usually there is only one call in it). 

Is there any way to query properties about the method being compiled (actually the caller) from codegen? If so, it would be better to use that instead of using flags.